tests/MongoDB.Driver.Tests/AuthenticationTests.cs

```diff
@@ -329,20 +355,57 @@ private void AssertAuthenticationFails(MongoClientSettings settings)
         {
             return CreateDatabaseUser(client, "admin", userName, password, "readWriteAnyDatabase", mechanisms);
         }
-        
+
         private BsonDocument CreateDatabaseUser(
             MongoClient client,
             string source,
-            string userName, 
-            string password, 
-            string role, 
+            string userName,
+            string password,
+            string role,
             params string[] mechanisms)
         {
             var createUserCommand =
-                $"{{createUser: '{userName}', pwd: '{password}'," 
+                $"{{createUser: '{userName}', pwd: '{password}',"
                 + $"roles: ['{role}'], mechanisms: {mechanisms.ToJson()}}}";
             return client.GetDatabase(source).RunCommand<BsonDocument>(createUserCommand);
         }
-        
+
+        private BsonDocument CreateX509DatabaseUser(MongoClient client, string userName)
+        {
+            var createUserCommand =
+                $"{{ createUser: '{userName}'," +
```

Comment:
Formatting like: `{ a : 'b' }` :)

Reply:
Ah bleh, I used the formatting from the MongoDB shell docs. Fixed :)